src/Environment.php

```diff
@@ -410,7 +410,7 @@ private function getRenderersByClass(array &$list, string $class, string $type):
             return $list[$class];
         }
 
-        while ($parent = \get_parent_class($parent ?? $class)) {
+        while (\class_exists($parent = $parent ?? $class) && $parent = \get_parent_class($parent)) {
```

Comment:
Kinda funky assignments, but this is the most compact and efficient way to do it I think?

Reply:
Works for me!